.travis.yml

```diff
@@ -1,11 +1,27 @@
 language: cpp
-
+sudo: required
+dist: trusty
+os:
+  - linux
+  - osx
 compiler:
   - clang
   - gcc
+rvm:
+  - 1.9.3
+  - 2.0
+  - 2.2
```

Comment:
it does not seem to work (always runs with 2.2.3), probably because it's a c++ project